code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -202,17 +191,20 @@
 	desc = "A flask of holy water...it's been sitting in the Necropolis a while though."
 	list_reagents = list("hell_water" = 100)
 
-/obj/item/reagent_containers/food/drinks/holyoil
+/obj/item/reagent_containers/food/drinks/bottle/holyoil
 	name = "flask of zelus oil"
 	desc = "A brass flask of Zelus oil, a viscous fluid with a scenting of brass - this flask may be sipped or thrown."
 	icon_state = "zelusflask"
-	list_reagents = list("holyoil" = 30) //Powerfull
+	list_reagents = list("holyoil" = 30)
 	volume = 30
 	foodtype = NONE
-	force = 12 //Same as a toolbox
+	force = 18
```

Comment:
They are not ment to be this powerful
12 was fine for weapons that costs 1 brass.

Reply:
bottle/smash() used to override the force to 15 anyway.

Reply:
Harm intent against objects is my fear with high numbers with this item as toolbox levels of harm was for smashing things.

Reply:
And? lit welders are already stronger than these.

Reply:
Also the bottle will still smash and become broken against objects too.